test/visual/test.js

```diff
@@ -12,14 +12,20 @@ gemini.suite('vaadin-progress-bar', rootSuite => {
       suite
         .setUrl(`default.html?theme=${theme}`)
         .setCaptureElements('#default-tests')
-        .capture('vaadin-progress-bar');
+        .capture('vaadin-progress-bar')
```

Comment:
This `capture` creates another folder with the same screenshots as in `default` (the next line).

Please remove this line, and the corresponding folders with screenshots it creates:

```
vaadin-progress-bar/default-tests-lumo/vaadin-progress-bar/chrome.png
vaadin-progress-bar/default-tests-material/vaadin-progress-bar/chrome.png
```

Reply:
done